settings.py

```diff
@@ -0,0 +1,74 @@
+
+import pygame
+from gi.repository import Gtk
+from button import Button
+
+
+class settings:
+
+    def run(self, gameDisplay, bg_dimensions, offset, config):
+        black = (0, 0, 0)
+        bg_color = (248, 237, 221)

```

Comment:
Why hardcode these?

Reply:
Is there any other way to do it?

Reply:
Yes, `Gdk.RGBA` that parses these colors and gets the rgb values if you need it, that makes it more modular.

Reply:
Oh yes, I will do so